docs/ReleaseNotes.md

```diff
@@ -1,6 +1,42 @@
+| ------------------------------- | ----------------------------------------------------------------------------------------------------------- | ------- | ----------- | ----------- |
+| [PX1060](diagnostics/PX1060.md) | DAC fields should be strongly typed to be used in fluent BQL queries.                                       | Message | Available   | Available   |
+| [PX1061](diagnostics/PX1061.md) | Constants should be strongly typed to be used in fluent BQL queries.                                        | Message | Available   | Available   |
+| [PX1090](diagnostics/PX1090.md) | `PXSetupNotEnteredException` cannot be thrown in action handlers.                                           | Warning | Available   | Unavailable |
+| [PX1091](diagnostics/PX1091.md) | This invocation of the base action handler can cause a `StackOverflowException`.                            | Warning | Available   | Unavailable |
+| [PX1092](diagnostics/PX1092.md) | Action handlers must be decorated with the `PXUIField` attribute and the `PXButton` attribute or its successors. | Error | Available   | Available |
+| [PX1093](diagnostics/PX1093.md) | In a graph declaration, the first type parameter of `PXGraph` must be the graph type.                       | Error   | Available   | Available   |
+| [PX1094](diagnostics/PX1094.md) | Every DAC should be decorated with the `PXHidden` or `PXCacheName` attribute.                               | Warning | Available   | Available   |
+| [PX1095](diagnostics/PX1095.md) | A field with the `PXDBCalced` attribute must have an unbound type attribute, such as `PXDate`, `PXDecimal`. | Error   | Available   | Unavailable |
+
+### Code Map
+In this version, you can navigate through the code of a graph by using the code map. The code map, which is shown in the following screenshot, displays the tree of data views and their delegates, actions and action handlers, and row and field events. Row events are grouped by the DACs to which they are related, field events are grouped by the DACs and DAC fields. By clicking the nodes of the tree, you can navigate between these elements in the code of the graph. If you switch between the files, the code map is updated automatically.
+
+![Code Map](images/CodeMap.png)
+
+The code map is an experimental feature, which is planned to be improved in the next versions. To open the code map for a graph, click **Code Map (Experimental)** on the **Acuminator** main menu of Visual Studio.
+
+### Fixed Bugs
+In this version of Acuminator, the following bugs have been fixed:
+
+ - The PX1030 diagnostic incorrectly worked with customizations of `PXDefault` on original DAC fields. Now the diagnostic can display either the error or the warning. For details, see the description of the diagnostic in [PX1030](diagnostics/PX1030.md).
```

Comment:
Could you please explain why only one bug from 13 is mentioned in release notes, and others marked as "excluded"? I'm just curious.

Reply:
@dropsonic , because other bugs were found internally. Usually only the bugs that are found externally are described in release notes. The described bug changed the behavior of the diagnostic therefore I described it in RN. If you think more bugs should be described, please specify which ones.

Reply:
@k-s-popova unfortunately, with Acuminator it is not so clear since our own developers are the end-users of our product, and we don't have support requests in the workflow. We often receive complains by Skype/email/etc. and after that file bug by ourselves. We'll think about adopting "How Found" field for this purpose to make it more clear.
For now, I removed `ExcludedFromReleaseNotes` from the bugs that, in my opinion, worth mentioning.

Reply:
all added